hybrid/financial/custom_financial_model.py

```diff
@@ -76,6 +76,65 @@ class SystemCosts(FinancialData):
     om_batt_replacement_cost: float
 
 
+@dataclass
+class Depreciation(FinancialData):
+    """
+    These financial inputs are used in simulate_financials in various files.
+    The names correspond to PySAM.Singleowner variables.
+    To add any additional system cost, first see if the variable exists in Singleowner, and re-use name.
+    This will simplify interoperability
+    """
+    depr_alloc_macrs_5_percent: float
+    depr_alloc_macrs_15_percent: float
+    depr_alloc_sl_5_percent: float
+    depr_alloc_sl_15_percent: float
+    depr_alloc_sl_20_percent: float
+    depr_alloc_sl_39_percent: float
+    depr_alloc_custom_percent: float
+    depr_bonus_fed_macrs_5: float
+    depr_bonus_sta_macrs_5: float
+    depr_itc_fed_macrs_5: float
+    depr_itc_sta_macrs_5: float
+    depr_bonus_fed_macrs_15: float
+    depr_bonus_sta_macrs_15: float
+    depr_itc_fed_macrs_15: float
+    depr_itc_sta_macrs_15: float
+    depr_bonus_fed_sl_5: float
+    depr_bonus_sta_sl_5: float
+    depr_itc_fed_sl_5: float
+    depr_itc_sta_sl_5: float
+    depr_bonus_fed_sl_15: float
+    depr_bonus_sta_sl_15: float
+    depr_itc_fed_sl_15: float
+    depr_itc_sta_sl_15: float
+    depr_bonus_fed_sl_20: float
+    depr_bonus_sta_sl_20: float
+    depr_itc_fed_sl_20: float
+    depr_itc_sta_sl_20: float
+    depr_bonus_fed_sl_39: float
+    depr_bonus_sta_sl_39: float
+    depr_itc_fed_sl_39: float
+    depr_itc_sta_sl_39: float
+    depr_bonus_fed_custom: float
+    depr_bonus_sta_custom: float
+    depr_itc_fed_custom: float
+    depr_itc_sta_custom: float
```

Comment:
while this is following the structure correctly, I don't think you need to include Incentives in your simple custom financial model, unless of course you know how to calculate them already and it's not hard / time-consuming.

Reply:
These are added because the Grid object currently uses the Singleowner model and there would otherwise be an error when it tries to aggregate the PV financial model parameters. I'll remove these later once I generalize the Grid financial model.

Reply:
Gotcha thanks